rho/host_discovery.py

```diff
@@ -138,9 +138,9 @@ def create_ping_inventory(vault, vault_pass, profile_ranges, profile_port,
     ansible_utils.log_yaml_inventory('Ping inventory', yml_dict)
 
     total_hosts_count = len(all_hosts)
-    rho_discovery_timeout = os.getenv('RHO_DISCOVERY_TIMEOUT', 5)
+    rho_discovery_timeout = os.getenv('RHO_DISCOVERY_TIMEOUT', 20)
```

Comment:
We should probably be checking that this value can be converted to an integer before casting it, and if not take the default value.
https://stackoverflow.com/a/1267145